packages/flutter/lib/src/material/card.dart

```diff
@@ -13,8 +13,8 @@ import 'material.dart';
 ///
 /// See also:
 ///
-///  * [Dialog]
-///  * [showDialog]
+///  * [ListItem], to display icons and text in a card.
+///  * [Dialog] and [showDialog], to display a modal card.
```

Comment:
I'd just link to `showDialog` and not to `Dialog` itself.  `showDialog` has good docs for people who want to show a dialog.